setup.py

```diff
@@ -29,6 +29,7 @@
         + extras_require['contrib']
         + extras_require['shellcomplete']
         + [
+            'scikit-hep-testdata>=0.4.10',
```

Comment:
I opened this PR before `scikit-hep-testdata` `v0.4.11` was released so everything will fail. Once it is though, everything should pass.